packages/shiki-twoslash/README.md

```diff
@@ -131,6 +131,8 @@ export interface TwoslashShikiOptions {
     includeJSDocInHover?: true;
     /** Instead of showing twoslash exceptions inline, throw the entire process like it will on CI */
     alwayRaiseForTwoslashExceptions?: true;
+    /** Ignore transforming certain code blocks */
+    ignore?: string[];
```

Comment:
I think `ignore` as a name is probably a bit too ambiguous on its own (e.g. could it be typescript related?) perhaps:

- `ignoreCodeblocksWithCodefenceMeta`
- `codefenceIgnore`

Reply:
Done.